css/index.css

```diff
@@ -34,14 +35,136 @@
   --navbar-menu-li-hover-background-color: #678684;
   --nav-container-font-family: poppins, sans-serif;
   --menu-link-font-family: roboto, sans-serif;
+  --details-top-position: 0;
+  --article-container-background: rgba(193 199 208 0.5);
+  --details-btn-hover-color: white;
+  --details-background-color: rgba(168 164 162 / 70%);
 }
 
 body {
   background-color: var(--body-background-color);
 }
 
-.d-none {
-  display: none;
+.close-details {
+  right: 4%;
+}
+
+.article-container {
+  background: var(--article-container-background);
+}
+
+a:hover path {
+  fill: var(--btn-active-background-color);
+}
+
+.details {
+  background-color: var(--details-background-color);
+  position: fixed;
+  inset: 0;
+}
+
+.details-works-article {
+  margin-top: 15px;
+  border-radius: 15px;
+  width: 90%;
+  padding: 1rem 2rem;
+  margin-left: 35px;
+  position: fixed;
```

Comment:
Define the margin elements together to avoid repetition

Reply:
Hi @ibukun53 Thanks for the review, but I don't think there's a need to create a margin, because I only worked with the margin-top and margin-left only in here

Reply:
What I mean is putting it like dis instead of splitting it
Margin: 8px 35px;